open-machine-learning-jupyter-book/assignments/ml-fundamentals/ml-linear-regression-2.ipynb

```diff
@@ -5,270 +5,300 @@
    "cell_type": "markdown",
```

Comment:
<p>The step 2 is not the same style as others</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>